pilot/pkg/security/authn/v1beta1/policy_applier_test.go

```diff
@@ -705,6 +704,15 @@ func TestJwtFilter(t *testing.T) {
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
 			istiotest.SetBoolForTest(t, &features.EnableRemoteJwks, c.enableRemoteJwks)
+			NewPolicyApplier("root-namespace", c.in, nil, push).JwtFilter()
+			for _, config := range c.in {
+				for _, jwtRule := range config.Spec.(*v1beta1.RequestAuthentication).JwtRules {
+					retryToCheckCache(5, 20*time.Millisecond, func() bool {
+						return push.JwtKeyResolver.CheckPubKeyExistInCache(jwtRule.Issuer, jwtRule.JwksUri)
+					})
```

Comment:
```suggestion
			retry.UntilOrFail(t, func() bool {
						return push.JwtKeyResolver.CheckPubKeyExistInCache(jwtRule.Issuer, jwtRule.JwksUri)
					})
```

and can update other methods + remove the retryToCheckCache function

Reply:
Actually its not clear what this function was doing originally... since we do not do anything if/when it times out?